src/lib/core/fetch/resolveCoreAccount.ts

```
@@ -0,0 +1,63 @@
+  umi: Umi,
+  id: string,
+  options: ResolveCoreAccountOptions = {},
+): Promise<ResolvedCoreAccount> {
+  const address = publicKey(id)
+
+  if (options.forceCollection) {
+    const collection = await safeFetchCollectionV1(umi, address).catch(() => null)
+    if (!collection) {
+      throw new Error(`Unable to fetch collection at address: ${id}`)
+    }
+    return { id, isCollection: true }
```

ESLint

[error] Expected blank line before this statement.

(padding-line-between-statements)